SwiftVideoBackground/Sources/VideoBackgroundError.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
```

Comment:
No idea how this was working before without this import lol

Reply:
😂😂